public/index.html

```diff
@@ -0,0 +1,171 @@
+                clip-rule="evenodd"
+              />
+            </svg>
+          </button>
+        </div>
+        <div class="contact-btn-container">
+          <ul class="contact-btn">
+            <li><a href="contact.html">お問い合わせ・資料請求</a></li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="hero container">
+        <img src="/public/assets/1.jpg" alt="hero carousel" />
+      </section>
+      <div class="features-main-heading section">
+        <h1 class="features-heading">今月の特集</h1>
+        <p class="features-subheading">This Month's Special</p>
+      </div>
+      <section class="feature container-feature container section">
+        <div class="feature-wrapper">
+          <div class="features-img-content">
+            <img class="features-img" src="/public/assets/1.jpg" alt="" />
+          </div>
+          <div class="features-content">
+            <h4 class="features-title">ここにタイトルを入れます</h4>
+            <p class="features-description">
+              ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="feature-list container-feature-list container section">
+        <div class="feature-list-main-heading">
+          <h1 class="feature-list-heading">施工事例</h1>
+          <p class="feature-list-subheading">Example of Construction</p>
+        </div>
+        <div class="feature-list-card-container container">
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/1.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-1">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/2.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">ここに説明を入れます。サンプルテキスト。</p>
+            <button class="card-btn btn-2">詳しくみる</button>
+          </div>
+          <div class="feature-list-img-card">
+            <img class="card-img" src="/public/assets/3.jpg" alt="house 1" />
+            <h3 class="card-title">東京都XX様宅</h3>
+            <p class="card-description">
+              ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。ここに説明を入れます。サンプルテキスト。
+            </p>
+            <button class="card-btn">詳しくみる</button>
+          </div>
+        </div>
+      </section>
+      <section class="commerce-info container-commerce container section">
+        <div class="whats-new">
+          <h1>新着情報</h1>
+          <hr />
+          <div class="commerce-info-row">
+            <time class="commerce-dates" datetime="2023-11-17">2023/11/17</time>
+            <div class="commerce-tags tag1">お知らせ</div>
```

Comment:
[should]
This one must be a heading tag. In this case, it must be `h2`.

Reply:
@sharray2918 
I can your point and I have changed it immediately!